build/commands/lib/build.js

```diff
@@ -41,7 +41,8 @@ const touchOverriddenFiles = () => {
 
   const chromiumSrcDir = path.join(config.srcDir, 'brave', 'chromium_src')
   var sourceFiles = util.walkSync(chromiumSrcDir, applyFileFilter)
-  const additionalGen = getAdditionalGenLocation()
+//   const additionalGen = getAdditionalGenLocation()
+  const additionalGen = undefined
```

Comment:
why do we need that?

Reply:
There is a bug when building release+x64, where the output folder will be `android_Release` in first run but npm will use `android_Release_x64` while resuming which makes the build starts over.